### PR TITLE
Add gltf-loader feature

### PR DESCRIPTION
**Pull request by alteous**
Addresses part of #156 .

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -52,3 +53,11 @@ glutin = { version = "0.12", optional = true }
 env_logger = "0.4"
 notify = "4"
 rand = "0.3"
+
+[[example]]
```
Comment (kvark) [inline Cargo.toml]: I'm curious. Does registering only some of the examples here still allows running the others? I.e. have you tried running other examples with this change?

File: src/animation.rs [hunk 2]
```diff
@@ -31,7 +31,7 @@
 //!
 //! First, we create a [`Mixer`] to play our animation.
 //!
-//! ```rust,no_run
+//! ```rust,no_run,ignore
```
Comment (kvark) [inline src/animation.rs]: why is `ignore` needed here? Perhaps, we need a TODO comment to rewrite those docs without the use of gtf-loader

File: Cargo.toml (hunk 1 above)
Comment (alteous) [reply Cargo.toml]: Oops. Nope, will fix.

File: src/animation.rs (hunk 2 above)
Comment (alteous) [reply src/animation.rs]: The TODOs would be difficult to notice.

```rust
//! ```rust,no_run,ignore
//! # // TODO: Rewrite test
//! ...
//! ```

Comment (alteous) [reply src/animation.rs]: Oh, perhaps just at the crate root instead?

File: Cargo.toml (hunk 1 above)
Comment (kvark) [reply Cargo.toml]: ouch, too bad I removed them. Maybe you can just grab that from my changeset to not rewrite by hand

File: src/animation.rs (hunk 2 above)
Comment (kvark) [reply src/animation.rs]: yep, that would do

**Comment by kvark** [comment]
bors r-
wait, we need to make sure it's CI tested

**Comment by kvark** [comment]
I suggest we add `--all-features` to `cargo test` while leaving `cargo build` untouched

**Comment by alteous** [comment]
> I suggest we add --all-features to cargo test while leaving cargo build untouched

Add to where? Travis CI?

**Comment by vitvakatu** [comment]
Yes

**Comment by alteous** [comment]
Should be good to go now.

**Comment by vitvakatu** [comment]
Looking good to me, merging
bors r+

**Pull request merged**